.github/workflows/ci.yml

```diff
@@ -342,10 +245,34 @@ jobs:
           image: oeffis/backend:${{ github.sha }}-untested-amd64
           ports:
             - 3000:3000
+          env:
+            VRR_BASE_URL: http://mocks-server:3100
+          # options: >-
+          #   --health-cmd "curl -L --fail http://localhost:3000"
+          #   --health-interval 10s
+          #   --health-timeout 5s
+          #   --health-retries 5
```

Comment:
here are still comments?

Reply:
Yes. I originally planned to enable health checks again at a later point, but skipped it for now. I will remove the comments.

Reply:
Done.